grow/pods/footnotes.py

```diff
@@ -0,0 +1,89 @@
+# -*- coding: utf-8 -*-
+"""Footnotes for documents using chicago manual style."""
+
+import collections
+import re
+
+SYMBOLS = [
+    '*',
+    u'†',
+    u'‡',
+    u'§',
+    u'||',
+    u'¶',
+    '#',
+]
+NUMERICAL_SYMBOLS = {
+    u'0': u'⁰',
+    u'1': u'¹',
+    u'2': u'²',
+    u'3': u'³',
+    u'4': u'⁴',
+    u'5': u'⁵',
+    u'6': u'⁶',
+    u'7': u'⁷',
+    u'8': u'⁸',
+    u'9': u'⁹',
+}
+NUMERIC_LOCALES_REGEX = re.compile(r'_(DE|CA)$', re.IGNORECASE)
```

Comment:
Nice! The "DE|CA" thing you're basing this on was a project-specific requirement.

Although I'm operating on good information that German sites should use numerical symbols by default, I would probably make this a little more configurable for users by moving this out into a constant such as `NUMERIC_TERRITORIES` so that people can override this on a per-project basis.